### ISSUE TITLE
Accessiblity issue: Elementor creates empty links on tabs widget

### ISSUE THREAD

**Issue by laxmariappan** [elementor/elementor#13191]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Elementor tabs create empty anchor tags on the tabs widget. This makes the link uncrawlable. 

Attaching the screenshots of the web.dev/measure/ reports and page urls

Live site: https://eternusglobal.com/
Test site: https://dev.eternusglobal.com/themes/tabs-test/

![report1](https://user-images.githubusercontent.com/22427070/100507350-50839700-31a8-11eb-9da3-2224e8a2a13f.png)

![tabs](https://user-images.githubusercontent.com/22427070/100507410-55484b00-31a8-11eb-9589-28571c5051bc.png)

![tabs-demo-site](https://user-images.githubusercontent.com/22427070/100507450-57aaa500-31a8-11eb-857d-95fe84bba7f7.png)

## Steps to reproduce

Add tabs (in my case vertical tabs) to a page or post

Publish it

Check the page on web.dev/measure/

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.6.42-84.2
	PHP Version: 7.4.9
	PHP Max Input Vars: 1000
	PHP Max Post Size: 20M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.3
	Site URL: https://eternusglobal.com
	Home URL: https://eternusglobal.com
	WP Multisite: No
	Max Upload Size: 20 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 8
	Debug Mode: Inactive

== Theme ==
	Name: Generate Press Child
	Version: 1.0.0
	Author: Kris
	Child Theme: Yes
	Parent Theme Name: GeneratePress
	Parent Theme Version: 3.0.2
	Parent Theme Author: Tom Usborne

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36

== Active Plugins ==
	Accelerated Mobile Pages
		Version: 1.0.69.1
		Author: Ahmed Kaludi, Mohammed Kaludi

	Captcha Code
		Version: 2.7
		Author: Vinoj Cardoza

	CF Post Formats
		Version: 1.3.1
		Author: crowdfavorite

	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	Elementor
		Version: 3.0.14
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.8
		Author: Elementor.com

	ElementsKit Lite
		Version: 2.0.9.3
		Author: Wpmet

	Head, Footer and Post Injections
		Version: 3.2.1
		Author: Stefano Lissa

	LiteSpeed Cache
		Version: 3.5.2
		Author: LiteSpeed Technologies

	Publish to Schedule
		Version: 4.4.2
		Author: Alex Benfica

	Rank Math SEO
		Version: 1.0.52.3
		Author: Rank Math

	Simple History
		Version: 2.37.2
		Author: Pär Thernström

	User Login History
		Version: 2.0.0
		Author: Er Faiyaz Alam

	Wordfence Security
		Version: 7.4.12
		Author: Wordfence

	WP-Sweep
		Version: 1.1.3
		Author: Lester 'GaMerZ' Chan


== Elements Usage ==
	
	archive : 2
		heading : 2
		posts : 2
		spacer : 8
	footer : 1
		button : 1
		divider : 1
		form : 1
		heading : 14
		icon-box : 3
		posts : 1
		spacer : 1
	header : 1
		button : 1
		ekit-nav-menu : 1
		image : 1
	page : 44
		animated-headline : 1
		button : 60
		divider : 15
		form : 5
		heading : 441
		icon : 196
		icon-box : 48
		icon-list : 25
		image : 85
		image-box : 5
		image-carousel : 1
		image-gallery : 1
		posts : 2
		spacer : 79
		text-editor : 339
	popup : 1
		form : 1
		heading : 1
		image : 1
	section : 14
		accordion : 2
		button : 38
		counter : 17
		divider : 25
		flip-box : 6
		form : 3
		heading : 185
		icon : 22
		icon-box : 4
		icon-list : 22
		image : 70
		image-box : 3
		price-table : 7
		progress : 4
		social-icons : 8
		spacer : 36
		testimonial : 14
		text-editor : 56
		video : 3
	single : 1
		form : 1
		heading : 1
		image : 1
		text-editor : 1
	widget : 0
		button : 18
	wp-post : 0
		button : 1
		elementskit-icon-box : 8
		elementskit-page-list : 21
		heading : 1
		tabs : 2
	wp-page : 41
		animated-headline : 1
		button : 30
		divider : 37
		form : 2
		google_maps : 1
		heading : 433
		icon : 187
		icon-list : 5
		image : 39
		image-box : 15
		image-carousel : 1
		posts : 1
		price-table : 4
		social-icons : 2
		spacer : 168
		template : 5
		text-editor : 309
	single-page : 2
		accordion : 4
		button : 12
		divider : 22
		flip-box : 12
		form : 4
		heading : 114
		icon : 32
		icon-box : 4
		image : 31
		image-box : 6
		price-table : 6
		social-icons : 5
		spacer : 16
		text-editor : 34
	error-404 : 1
		heading : 3
		icon-list : 4
		image : 1
		spacer : 7


== Log ==
	
JS: showing 19 of 19JS: 2020-08-26 07:29:32 [error X 20][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.1:2:469692] Cannot read property \'document\' of undefined 
JS: 2020-08-26 08:20:57 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.1:2:469677] this.container is undefined 
JS: 2020-09-01 01:22:12 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5:22:19954] View (cid: \"view1380\") has already been destroyed and cannot be used. 
JS: 2020-10-07 01:31:35 [error X 12][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.0.9:2:51490] Cannot read property \'[object Array]\' of undefined 
JS: 2020-10-07 01:41:13 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5:22:19954] View (cid: \"view1410\") has already been destroyed and cannot be used. 
JS: 2020-10-23 07:39:20 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5:22:19954] View (cid: \"view3034\") has already been destroyed and cannot be used. 
JS: 2020-10-26 06:35:08 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor-pro/assets/js/frontend.min.js?ver=3.0.5:2:46558] Failed to execute \'observe\' on \'IntersectionObserver\': parameter 1 is not of type \'Element\'. 
JS: 2020-10-26 06:50:31 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:2:22] Failed to execute \'querySelector\' on \'Document\': 1 argument required, but only 0 present. 
JS: 2020-10-26 06:50:34 [error X 7][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:2:36] Unexpected identifier 
JS: 2020-10-26 06:50:46 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:1] s is not defined 
JS: 2020-10-26 06:50:46 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:1] sc is not defined 
JS: 2020-10-26 06:50:47 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:1] scro is not defined 
JS: 2020-10-26 06:50:56 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:12] Failed to execute \'addEventListener\' on \'EventTarget\': 2 arguments required, but only 0 present. 
JS: 2020-10-26 06:50:59 [error X 4][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:12] Failed to execute \'addEventListener\' on \'EventTarget\': 2 arguments required, but only 1 present. 
JS: 2020-10-26 06:51:03 [error X 2][https://dev.eternusglobal.com/eternusv3/wp-admin/post.php?post=9606&action=elementor#:4:39] Unexpected token \')\' 
JS: 2020-10-26 06:57:49 [error X 4309][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:16820] Cannot read property \'clone\' of null 
JS: 2020-10-26 06:57:52 [error X 78][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:21012] Cannot read property \'changestop\' of null 
JS: 2020-10-28 08:38:22 [error X 7][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.12:2:204614] Cannot read property \'data\' of undefined 
JS: 2020-10-29 06:38:52 [error X 1][https://dev.eternusglobal.com/eternusv3/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.0.12:2:60285] Cannot read property \'attributes\' of undefined 

PHP: showing 4 of 4PHP: 2020-08-26 15:48:22 [notice X 1][/home/ausdigit2129/public_html/dev/eternusv3/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::217] Undefined index: post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-10-28 14:22:05 [notice X 1][/home/ausdigit2129/public_html/dev/eternusv3/wp-content/plugins/elementor/includes/elements/column.php::1007] Undefined index: _column_size [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-17 13:17:42 [notice X 2][/home/ausdigit2129/public_html/eternusglobal.com/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-27 15:43:16 [error X 1][/home/ausdigit2129/public_html/eternusglobal.com/wp-content/plugins/elementor/includes/base/widget-base.php::154] Allowed memory size of 134217728 bytes exhausted (tried to allocate 81920 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 202020-09-18 09:15:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.0',
  'to' => '3.0.4',
)]
2020-10-20 13:51:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.4',
  'to' => '3.0.5',
)]
2020-11-13 15:39:42 [info] elementor::elementor_updater Started 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Start  
2020-11-13 15:39:42 [notice] General Settings already exist. nothing to do. 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Finished 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Start  
2020-11-13 15:39:42 [notice] System colors already exist. nothing to do. 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Finished 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Start  
2020-11-13 15:39:42 [notice] Custom colors already exist. nothing to do. 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Finished 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Start  
2020-11-13 15:39:42 [notice] System typography already exist. nothing to do. 
2020-11-13 15:39:42 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Finished 
2020-11-13 15:39:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.1',
  'to' => '3.0.12',
)]
2020-11-16 19:05:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.12',
  'to' => '3.0.13',
)]
2020-11-16 19:15:01 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.5',
  'to' => '3.0.6',
)]
2020-11-27 11:45:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.13',
  'to' => '3.0.14',
)]
2020-11-27 11:45:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.6',
  'to' => '3.0.8',
)]


				

```
</details>

**Comment by shilo-ey** [comment]
Thanks!

Do you think this change resolves the issue? #13165

**Comment by shilo-ey** [comment]
Please see the fix we suggest in our Developer Edition latest version.

See more here: https://wordpress.org/plugins/elementor-beta

**Comment by DorShahar** [comment]
This issue has been resolved in Elementor v3.1.0
Feel free to update
Thanks!
